assets/js/admin/course-general-sidebar.js

```diff
@@ -171,6 +172,24 @@ const CourseGeneralSidebar = () => {
 				/>
 			) : null }
 
+			<HorizontalRule />
+
+			<h3>{ __( 'Enrollment', 'sensei-lms' ) }</h3>
+			<CheckboxControl
+				label={ __( "Don't allow self enrollment", 'sensei-lms' ) }
+				checked={ senfEnrollmentNotAllowed }
+				onChange={ ( checked ) =>
+					setMeta( {
+						...meta,
+						_self_enrollment_not_allowed: checked,
+					} )
+				}
+				help={ __(
+					"Students can't take this course by themselves. They can be enrolled only by teachers, administrators, or invitation.",
```

Comment:
What about "Students need to be manually enrolled."? I don't think we should mention invitation since that's a Pro feature.

Reply:
I used "Students need to be manually enrolled by teachers or administrators.". WDYT?
I thought to use "through WP Admin" instead of "by teachers or administrators", but I thought it could be confusing for invites.